src/App.js

```diff
@@ -76,34 +76,61 @@ export default function App() {
 
   return (
     <QueryClientProvider client={new QueryClient()}>
+      <SidePanel
+        width={400}
+        variant="hyproof"
+        heading="Certificate View"
+        title={persona.name}
+        callback={(e) => setShowSelector(e.isOpen)}
+      >
+        {personas.map((el) => (
+          <SidePanel.Item
+            {...el}
+            key={el.id}
+            active={el.id === current}
+            update={(_, persona) => handlePersonaSwitch(persona)}
+            variant="hyproof"
+          />
+        ))}
+      </SidePanel>
       <FullScreenGrid
         color={persona.background}
         showSelector={showSelector}
         areas={[
-          ['home', 'nav', 'nav'],
-          ['header', 'header', 'sidebar'],
-          ['timeline', 'main', 'sidebar'],
+          ['home', 'nav'],
+          ['header', 'header'],
+          ['timeline', 'main'],
+          ['timeline', 'sidebar'],
+        ]}
+        columns={['minmax(min-content, 1fr)', '2fr']}
+        rows={['78px', '98px', '1fr', 'auto']}
+        byWidth={[
+          {
+            minWidth: 1000,
+            areas: [
+              ['home', 'nav', 'nav'],
+              ['header', 'header', 'sidebar'],
+              ['timeline', 'main', 'sidebar'],
```

Comment:
How does it work, like a sub element, might be a nice to get a walkthrough if I won't be able to work out after rebasing. Still a habit is to fallback to flex as can do from top of my head

Reply:
if you look at the grid component in the ui library there's a few good examples. The `minWidth` fields use a media query to set different values. At the moment the `1500` minwidth one  is a bit magic value like to stop things getting too big. I am still looking for a better way